.gitignore

```diff
@@ -12,9 +12,11 @@ realm-object-store.xcodeproj/
 
 # Build products
 src/librealm-object-store.a
+src/librealm-object-store-static.a
```

Comment:
This shouldn't be necessary after 2412932253f25ad6c19ef6ba8c5273d7e101bd24 (late August).